pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/data/repository/UserRepository.java

```diff
@@ -94,6 +90,7 @@ public User findOrCreate(User inputUser) {
             user = createUser(inputUser);
         } catch (NonUniqueResultException e) {
             logger.error("findOrCreate() " + e.getClass().getSimpleName() + ": " + e.getMessage());
+            user = createUser(inputUser);
```

Comment:
Are we using this code for both open and auth? If yes, do we want this to create a user in auth if it's not unique by subject?